docs/templates/index.md

```diff
@@ -0,0 +1,12 @@
+---
+---
+
+When writing docs, start with one of the five templates we provide:
```

Comment:
Do we need to put these docs in any special path so they appear prominently on the site?